### DISCUSSION TITLE
a quick test of  fntsample

### DISCUSSION THREAD

**Discussion by jsbien** [psb1558/Junicode-font#49] [General]
Produced with https://bitbucket.org/jsbien/fntsample-fork-with-ucd-comments (no comments yet)
[test.pdf](https://github.com/psb1558/Junicode-New/files/5710311/test.pdf)
And with standard Unicode comments (https://bitbucket.org/jsbien/unicode-ucd-parser/):
[test1.pdf](https://github.com/psb1558/Junicode-New/files/5710382/test1.pdf)
And with tentative plane 15 comments:
[test2.pdf](https://github.com/psb1558/Junicode-New/files/5710633/test2.pdf)
And plane 15 fully commented, but the output manifests some limitations of the program :-(
[plane15.pdf](https://github.com/psb1558/Junicode-New/files/5715124/plane15.pdf)

**Comment by psb1558** [comment]
This is incredible. What a great program!

**Reply by psb1558** [reply]
I wonder if it would be possible to pull from the MUFI database to annotate the rest of the PUA.

**Comment by jsbien** [comment]
I asked a question about exporting data from the MUFI database several years ago, but got no answer. I will have a look at http://www.kreativekorp.com/charset/PUADATA/PUBLIC/MUFI/, it seems useful for this purpose. Just used for a quick and dirty test:
[PUA.pdf](https://github.com/psb1558/Junicode-New/files/5719682/PUA.pdf). The data used are available at https://github.com/jsbien/unicode4polish.
The program is unfortunately orphaned and has  bugs/features which I'm unable to handle: https://github.com/jsbien/fntsample-with-comments/issues. Some help of somebody with at least a rudimentary knowledge of C would be highly useful.

**Comment by jsbien** [comment]
A quick and dirty update for  _horned o_:
[Junius-Regular_21-01-19_plane15.pdf](https://github.com/psb1558/Junicode-New/files/5840758/Junius-Regular_21-01-19_plane15.pdf)

**Reply by psb1558** [reply]
It looks good!
